packages/docs-page/index.tsx

```diff
@@ -127,7 +145,22 @@ export function DocsPageWrapper({
   )
 }
 
-export default function DocsPage({
+export interface DocsPageProps {
+  product: { name: string; slug: string }
+  baseRoute: string
+  showEditPage: boolean
+  additionalComponents: MDXProviderComponentsProp
+  staticProps: {
+    mdxSource: any // TODO: import { MDXRemoteProps } from "next-mdx-remote"
```

Comment:
I was thinking to do `Omit<MDXRemoteProps, "components">`

But we'll need to `export` it from https://github.com/hashicorp/next-mdx-remote/blob/main/src/index.tsx#L26

Reply:
See this section of the docs, the `MDXRemoteSerializeResult` import should be the type we want here. https://github.com/hashicorp/next-mdx-remote#typescript

Reply:
Addressed in https://github.com/hashicorp/react-components/pull/337/commits/a25c71a0b9b00298235785c04d4e10e095b7f63f